test/blackbox-tests/test-cases/sandboxing.t/run.t

```diff
@@ -260,7 +253,6 @@ Work around the above problem by using the absolute path [$PWD/../link].
 Sandboxed actions can observe external dependencies as symbolic links.
 
   $ dune build t --sandbox hardlink
-  symbolic link
```

Comment:
I guess this line got removed because Dune didn't rebuild the action. Could you force it to do that?